src/lp.jl

```diff
@@ -0,0 +1,174 @@
+#=
+Interface to lrs_solve_lp
+
+=#
+
+
+mutable struct LRSLinprogSolution
+    status::Symbol  # :Optimal, :Infeasible, :Unbounded, :UserLimit, or :Error
+    objval::Union{Nothing,Rational{BigInt}}
+    sol::Vector{Rational{BigInt}}
+    attrs::Dict
+end
+
+
+function setobj(P::Ptr{Clrs_dic}, Q::Ptr{Clrs_dat},
+                c::Vector{<:Real}, maximize::Bool)
```

Comment:
What do you think of changing `<:Real` to `<:Union{Rational, Integer}` ? If `c` is a vector of `Float64`, the user will get a bad surprise and will prefer a method error IMO.